lib/term_statistics.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+class TermStatistics
+  # TODO: reduce number of arguments
+  def initialize(term:, org_lookup:, people:)
```

Comment:
Hmmm. After looking a little closer, I don't think `people` is obvious here either. I would naturally have assumed that that was going to be a list of `EveryPolitician::Person` objects, but nope!